pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"gotest.tools/assert"
```

Comment:
I'd prefer not to add a dependency on `assert`-like libraries, since they don't always make the code any more readable, and developers end up having to learn the particular behaviors/quirks of the assert library. Pair that with the fact that they're not uniformly used across the codebase, and I think they end up causing more headaches than they solve.

Reply:
Makes sense. Shall check what's being currently used for similar in this codebase

Reply:
also, it should have been `gotest.tools/v3/assert` :upside_down_face: 
The real value of `gotest.tools` is the other packages (fs, env, golden, icmd, …) — *says one of the other of the lib* :grin:.

Reply:
Has been removed 😄